src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -307,15 +296,20 @@ impl Renderer for HtmlHandlebars {
         fs::create_dir_all(&destination)
             .chain_err(|| "Unexpected error when constructing destination path")?;
 
-        for (i, item) in book.iter().enumerate() {
+        let mut depthfirstiterator = book.iter();
+        let mut is_index = true;
+        while let Some(item) = depthfirstiterator.next() {
```

Comment:
Shouldn't this just be a plain old `for` loop?

Reply:
Yeah, I'm not sure why this was changed. This is from the original PR.